src/triage/cli.py

```diff
@@ -48,18 +50,20 @@ def __init__(self, parser):
         )
         parser.add_argument(
             '-s', '--setup',
-            help="file path to Python module to import before running the Experiment",
+            help=f"file path to Python module to import before running the "
+                 f"Experiment (default: {self.SETUP_FILE_DEFAULT})",
         )
 
     def setup(self):
-        setup_path = self.args.setup or os.path.abspath('experiment.py')
-        if setup_path is not None:
-            logging.info(f"Loading configurations from {setup_path}")
-            spec = importlib.util.spec_from_file_location("triage_config", setup_path)
-            triage_config = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(triage_config)
-            logging.info(f"Configuration loaded")
-        return None
+        if not self.args.setup and not os.path.exists(self.SETUP_FILE_DEFAULT):
+            return
+
+        setup_path = self.args.setup or self.SETUP_FILE_DEFAULT
+        logging.info("Loading setup module at %s", setup_path)
```

Comment:
perhaps less of a big deal now with `f""` strings. but, the API of the logging module is the above. it only stringifies and interpolates arguments if the log level is set such that the message needs to be generated.